src/app/import/components/helpers/upload-helper.ts

```diff
@@ -0,0 +1,72 @@
+import { Router } from "@angular/router";
+import { timer } from "rxjs";
+import { ExistingProjectUploadHelper, LookupListsUploadHelper, RecordAddUploadHelper, RecordNewUploadHelper } from "./upload-specific";
+import { UploadFileType } from "./upload-types";
+import { UploadComponent } from "../upload/upload.component";
+
+export class UploadHelper {
+    baseComponent: UploadComponent;
+    recordNewUploadHelper: RecordNewUploadHelper;
+    recordAddUploadHelper: RecordAddUploadHelper;
+    existingProjectUploadHelper: ExistingProjectUploadHelper;
+    lookupListsUploadHelper: LookupListsUploadHelper;
+
+    constructor(baseComponent?: UploadComponent, recordNewUploadHelper?: RecordNewUploadHelper, recordAddUploadHelper?: RecordAddUploadHelper, existingProjectUploadHelper?: ExistingProjectUploadHelper, lookupListsUploadHelper?: LookupListsUploadHelper) {
```

Comment:
im not sure i understand this correctly but the constructor of the UploadHelper wants e.g. an RecordAddUploadHelper. The RecordAddUplaodHelper has in the constructor an uploadHelper variable that is created with new.

Doesn't seem right to me